CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 - Added curve dimensions option to the Separate Text node.
 - Added matrices output in *Mesh Points Scatter* node.
 - Added edges mode in *Mesh Points Scatter* node.
+- Added *Vertex Color Fill* in *GP Stroke Info*, *GP Stroke From Points*,
+  and *Set GP Stroke Attributes* nodes.
```

Comment:
`Added *Vertex Color Fill* attribute to GP strokes.`

Reply:
Done.